terraform/kevsync_failure_alarms.tf

```diff
@@ -48,7 +56,7 @@ resource "aws_cloudwatch_metric_alarm" "kevsync_failure" {
     id = replace("kevsync_failure_count_${each.value.hostname}", ".", "_")
     metric {
       dimensions = {
-        InstanceId = each.key
+        InstanceId = each.value
```

Comment:
This is not correct- https://github.com/cisagov/cyhy_amis/pull/423/commits/92dc671a86b3431a33cd290b641740cb938ce274 should be reverted, since `local.db_instances` is a map, not a list.

Reply:
I rebased, removing the commit.